src/styles/blocks/page.scss

```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  &__budy {
```

Comment:
There seems to be a typo in the class name '&__budy'. It should likely be '&__body' if you are referring to the body of the page.